tests/virtualfolder_helpers.py

```diff
@@ -60,3 +63,6 @@ def test_extract_vfolder_from_path():
     # locations, but the VirtualFolderTreeItem pootle_path is unique, thus only
     # one exists.
     assert 1 == VirtualFolderTreeItem.objects.filter(pootle_path=path).count()
+
+    vfolder0.delete()
```

Comment:
They were necessary in my local install since otherwise test crashed. Will keep them.